pom.xml

```diff
@@ -48,7 +48,7 @@
         <reportingVersion>0.10.4</reportingVersion>
         <providermanagementVersion>2.5.0</providermanagementVersion>
         <idgenVersion>4.4.0</idgenVersion>
-        <uicommonsVersion>2.2.0</uicommonsVersion>
+        <uicommonsVersion>2.2-SNAPSHOT</uicommonsVersion>
```

Comment:
@dkayiwa Issues with not updating local copy from upstream so it works locally - great catch!!